pkg/install/v1alpha1/cstor_volume_0.7.0.go

```diff
@@ -132,6 +132,10 @@ spec:
   output: cstor-volume-list-output-default-0.7.0
 ---
 # runTask to list cvrs if this is a clone volume
+# this run task will list all the cvrs that belong to
+# the source volume. If this is not a cloned volume
```

Comment:
Good. Can we be more clear.

Can we write "This RunTask is meant to be run only during clone volume create requests. However, clone & volume creation follow the same CASTemplate specifications. As of today, RunTask can not be run based on conditions. Hence, it contains a logic which will list empty pools "